cloudinit/sources/DataSourceAzure.py

```diff
@@ -344,6 +344,7 @@ class DataSourceAzure(sources.DataSource):
         EventType.BOOT,
         EventType.BOOT_LEGACY
     }}
+    excluded_drivers = BLACKLIST_DRIVERS  # Drivers excluded from network cfg
```

Comment:
The PR after this will address non-functional renames of "BLACKLIST_DRIVERS" to "excluded_drivers" throughout cloudinit. I didn't want to address this renaming in this branch as it would be much broader than this azure-specific change (and will touch distros/networking, cloudinit.net.__init__.py docs and potentially some config modules.